client/src/pages/SignUp.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
```

Comment:
This style and the one in LogIn are very similar. You could declare it in a separate file( e.g: AuthStyle.jsx) and import it on both components

Reply:
Sure.